03_Modules/Tenant/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@citrineos/tenant",
+  "version": "1.3.0",
```

Comment:
version is 1.3.0 to match release candidate, but if there's a separate version convention i can update, let me know!